### PR TITLE
Created TrueID presence

### PR DESCRIPTION
**Pull request by Faelayis**
Created TrueID presence

<details>
<summary>
Preview
</summary>
<br>

image
image
image
image
image

</details>

### CODE REVIEW

File: websites/T/TrueID/presence.ts [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+    delete presenceData.endTimestamp;
+  }
+  if (privacy) {
+    delete presenceData.state;
+    delete presenceData.buttons;
+  }
+  if (!presenceData.details) {
+    presence.setTrayTitle();
+    presence.setActivity();
+  } else presence.setActivity(presenceData);
```
Comment (Slowlife01) [inline websites/T/TrueID/presence.ts:167]:
```suggestion
  if (!presenceData.details) presence.setActivity();
  else presence.setActivity(presenceData);
```

**Review by Slowlife01 — CHANGES_REQUESTED** [review]

File: websites/T/TrueID/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  "url": [
+    "home.trueid.net",
+    "movie.trueid.net",
+    "tv.trueid.net"
+  ],
+  "description": {
+    "en": "True ID experience a new entertainment center more than and better privileges For every lifestyle, easy to use, complete in one place.",
+    "th": "ทรูไอดี สัมผัสประสบการณ์ใหม่ของศูนย์รวมความบันเทิงที่สุดกว่า และสิทธิพิเศษที่ดีกว่า เพื่อทุกไลฟ์สไตล์ ใช้งานง่าย จบครบในที่เดียว."
+  },
+  "iframe": false,
```
Comment (Slowlife01) [inline websites/T/TrueID/dist/metadata.json:28]:
```suggestion
```

**Review by Slowlife01 — DISMISSED** [review]

File: websites/T/TrueID/presence.ts [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+  if (!time) {
+    delete presenceData.startTimestamp;
+    delete presenceData.endTimestamp;
+  }
+  if (privacy) {
+    delete presenceData.state;
+    delete presenceData.buttons;
+  }
+  if (!presenceData.details) presence.setActivity();
+  else presence.setActivity(presenceData);
```
Comment (EncryptedDev) [inline websites/T/TrueID/presence.ts:165]: #4946 will enforce all presences to use this format
```suggestion
  if (presenceData.details) presence.setActivity(presenceData);
  else presence.setActivity();
```

File: websites/T/TrueID/presence.ts [hunk 4]
```diff
@@ -0,0 +1,166 @@
+const presence = new Presence({
+    clientId: "912784051801301053"
+  }),
+  browsingStamp = Math.floor(Date.now() / 1000);
```
Comment (EncryptedDev) [inline websites/T/TrueID/presence.ts:4]:
```suggestion
  browsingTimestamp = Math.floor(Date.now() / 1000);
```

File: websites/T/TrueID/presence.ts [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+
+presence.on("UpdateData", async () => {
+  const time = await presence.getSetting("timestamps"),
+    privacy = await presence.getSetting("privacy"),
+    buttons = await presence.getSetting("buttons"),
+    path = document.location,
+    lang = path.pathname.substring(1, 6),
+    presenceData: PresenceData = {
+      largeImageKey: "index",
+      startTimestamp: browsingStamp
```
Comment (EncryptedDev) [inline websites/T/TrueID/presence.ts:14]:
```suggestion
      startTimestamp: browsingTimestamp
```

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**